.github/workflows/validate_kong_image_trigger_via_label.yaml

```diff
@@ -79,17 +79,14 @@ jobs:
     needs:
     - extract-and-parse-container-image
     runs-on: ubuntu-latest
-    env:
-      KONG_MAJOR_VERSION: ${{ env.TEST_KONG_IMAGE_MAJOR_VERSION }}
-      KONG_MINOR_VERSION: ${{ env.TEST_KONG_IMAGE_MINOR_VERSION }}
     steps:
      - name: Choose KIC image and branch
        id: choose_kic_branch
        run: |
-          if [[ $KONG_MAJOR_VERSION -le 2 ]]; then
+          if [[ $TEST_KONG_IMAGE_MAJOR_VERSION -le 2 ]]; then
```

Comment:
These are set in a separate job so I'm surprised this works given that:

https://github.com/orgs/community/discussions/56849

says that:

> Variables set in `GITHUB_ENV` apply only to the current job. To carry the value over to another job, you need a [job output](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idoutputs). Mind that you need to define the job output based on the step output.

Do you have an exemplar CI job logs which prove that this works?

Reply:
Fixed these.